sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/FileSystemCertificates.java

```diff
@@ -0,0 +1,183 @@
+    public void setCertificateEntry(String filePath, String alias, Certificate certificate) {
+        List<String> allAlias = fileSystemAlias.values()
+                                               .stream()
+                                               .flatMap(l -> l.stream())
+                                               .collect(Collectors.toList());
+        if (!allAlias.contains(alias)) {
+            if (fileSystemAlias.get(filePath) == null) {
+                fileSystemAlias.put(filePath, new ArrayList<>());
+            }
+            fileSystemAlias.get(filePath).add(alias);
+            certificates.put(alias, certificate);
+        }
+    }
+
+
+    private X509Certificate getCertificateByAliasAndStream(InputStream inputStream, String filename, String alias) throws IOException {
+        X509Certificate certificate = null;
+        if (alias != null) {
+            byte[] bytes = KeyVaultKeyStore.readAllBytes(inputStream);
+            try {
+                CertificateFactory cf = CertificateFactory.getInstance("X.509");
+                certificate = (X509Certificate) cf.generateCertificate(
+                    new ByteArrayInputStream(bytes));
+            } catch (CertificateException e) {
+                LOGGER.log(WARNING, "Unable to load file system certificate from: " + filename, e);
+            }
+        }
+        return certificate;
+    }
+
+    void loadCertificatesFromFileSystem() {
+        try {
+            for (String filePath: certPaths) {
+                String[] filenames = getAbsoluteFilename(filePath);
+                if (filenames.length > 0) {
+                    for (String filename : filenames) {
+                        try (InputStream inputStream = new FileInputStream(new File(filePath + filename))) {
+                            String alias = filename;
+                            if (alias != null) {
+                                if (alias.lastIndexOf('.') != -1) {
+                                    alias = alias.substring(0, alias.lastIndexOf('.'));
+                                }
+                                X509Certificate certificate = getCertificateByAliasAndStream(inputStream, alias, filename);
+                                if (certificate != null) {
+                                    setCertificateEntry(filePath, alias, certificate);
+                                    LOGGER.log(INFO, "Load file system certificate: {0} from: {1}",
+                                        new Object[]{alias, filename});
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        } catch (IOException ioe) {
+            LOGGER.log(WARNING, "Unable to determine certificates to file system", ioe);
+        }
+    }
+
+    /**
+     * Get the filenames.
+     *
+     * @param path the path.
+     * @return the filenames.
+     */
+    private String[] getAbsoluteFilename(String path) {
```

Comment:
what  this `path` argument is expected? a path to the directory where cert files located?
does this method support sub directory?
what if a file path is given?

could you add more javadoc to the method about these info?

and something check is missing i think, for example:

1. if the path is not exist?
2. if the path is not readable?